core/src/main/java/org/mifos/mobilewallet/core/domain/model/CheckBoxStatus.java

```diff
@@ -0,0 +1,27 @@
+package org.mifos.mobilewallet.core.domain.model;
+
+public class CheckBoxStatus {
+
+    private String status;
```

Comment:
not sure what status signifies here. variable name shouldn't be `status`. Can be changed to simply be `text`